.circleci/config.yml

```diff
@@ -75,7 +75,7 @@ jobs:
     environment:
       # Alaska time (arbitrary timezone to test date logic)
       TZ: "America/Anchorage"
-    parallelism: 12
+    parallelism: 4
```

Comment:
Is this related to the changes in this PR?

Reply:
Not really. Since other containers finish earlier, I made this adjustment so that this container also finishes around the same time and catches potential errors earlier.